src/server/pkg/worker/api_server.go

```diff
@@ -224,6 +227,9 @@ func (a *APIServer) downloadData(logger *taggedLogger, inputs []*Input, puller *
 
 // Run user code and return the combined output of stdout and stderr.
 func (a *APIServer) runUserCode(ctx context.Context, logger *taggedLogger, environ []string) error {
+	defer func(start time.Time) {
+		logger.Logf("running user code for environment (%v) took %v\n", environ, time.Since(start))
+	}(time.Now())
```

Comment:
Yea, I was looking for a unique identifier to provide in the logs so that you could tie it to a job/datum ... but now that I think about it that's kind of the whole point of the logging interface we have setup (it adds that metadata). I'll remove the environ / inputs from the calls.